### PR TITLE
Mobile Portfolio pull request  number 2

### PR DESCRIPTION
**Pull request by Ashuaks**
The second milestone of the project "Portfolio setup and mobile version skeleton".
In this, I created the middle section of the Figma portfolio template 1.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,7 +27,108 @@ <h1>I'm Egbe Bless Ashuakie!</h1>
                 <li><a href=""><img src="images/rabbit.png" alt=""></a></li>
             </ul>
         </div>
-        
+        <div class="cards">
```
Comment (danmainah) [inline index.html:30]: - [x] Kindly ensure your work section is wrapped using the `<section>` tag, currently you are using a `div`.

File: index.html [hunk 2]
```diff
@@ -29,7 +27,108 @@ <h1>I'm Egbe Bless Ashuakie!</h1>
...
+                    <li><img src="images/small-circle.png" alt=""></li>
+                    <li><p>2015</p></li>
+                </ul>
+                <p class="daily-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="langs">
+                    <li><p>html</p></li>
+                    <li><p>css</p></li>
+                    <li><p>javaScript</p></li>
+                </ul>
+                <button type="button" class="btn see-projects">See Projects</button>
```
Comment (danmainah) [inline index.html:47]: - [x] Kindly add a hover effect on your buttons, ensure to use the recommended style below is a snippet of the format:

image

File: index.html [hunk 3]
```diff
@@ -29,7 +27,108 @@ <h1>I'm Egbe Bless Ashuakie!</h1>
...
+                <p class="daily-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="langs">
+                    <li><p>html</p></li>
+                    <li><p>css</p></li>
+                    <li><p>javaScript</p></li>
+                </ul>
+                <button class="btn see-projects" type="button">See Projects</button>
+            </div>
+        </div>
+        <div class="about_myself_section">
```
Comment (danmainah) [inline index.html:104]: - [x] Kindly ensure your about section is wrapped using a `section` tag, please replace the div tag with the section tag.

File: index.html [hunk 4]
```diff
@@ -29,7 +27,108 @@ <h1>I'm Egbe Bless Ashuakie!</h1>
...
+            <div class="about_myself">
+                <h2>About Myself</h2>
+                <p class="about_p">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <p class="about_lets">LETS CONNECT</p>
```
Comment (danmainah) [inline index.html:108]: - [x] Kindy use the `hx` tag for this line because let's connect is supposed to be a title.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Ashuaks , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 3 above)
Comment (wuletawwonte) [inline index.html:104]: - [ ] [OPTIONAL] Please wrap each section in `section` tag instead of `div` tags. That would make the code semantically correct.

**Review by wuletawwonte — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @Ashuaks  👋🏿👋🏿👋🏿, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 

approve

## TO HIGHLIGHT: 
- Your project is professional ✔️ 
- Correct Github Flow ✔️
- Your PR has a proper title and description ✔️
- Well structured files ✔️ 

## Required Changes ♻️
_Please check the comments under the review as well._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**